paimon-core/src/main/java/org/apache/paimon/operation/metrics/CompactionMetrics.java

```diff
@@ -19,21 +19,34 @@
 package org.apache.paimon.operation.metrics;
 
 import org.apache.paimon.annotation.VisibleForTesting;
-import org.apache.paimon.metrics.Histogram;
+import org.apache.paimon.data.BinaryRow;
 import org.apache.paimon.metrics.MetricGroup;
 import org.apache.paimon.metrics.MetricRegistry;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.DoubleStream;
+import java.util.stream.LongStream;
+
 /** Metrics to measure a compaction. */
 public class CompactionMetrics {
 
-    private static final int HISTOGRAM_WINDOW_SIZE = 100;
     private static final String GROUP_NAME = "compaction";
 
+    public static final String MAX_LEVEL0_FILE_COUNT = "maxLevel0FileCount";
+    public static final String AVG_LEVEL0_FILE_COUNT = "avgLevel0FileCount";
+    public static final String MAX_COMPACTION_THREAD_BUSY = "maxCompactionThreadBusy";
```

Comment:
Each task has only one backend thread, so according to this statement, busy should only have one metric.